recipes/libjpeg/all/test_package/conanfile.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        with tools.environment_append(RunEnvironment(self).vars):
```

Comment:
why `environment_append` with you are using `run_environment`  ?

Reply:
it was leftover from bincrafters version